google-datacatalog-rdbms-connector/tests/google/datacatalog_connectors/rdbms/scrape/metadata_scraper_test.py

```diff
@@ -25,11 +25,11 @@
 
 class MetadataScraperTestCase(unittest.TestCase):
     __MODULE_PATH = os.path.dirname(os.path.abspath(__file__))
+    __NORMALIZER_PATH = 'metadata_normalizer.MetadataNormalizer.normalize'
```

Comment:
```suggestion
    __NORMALIZER_CLASS = '{}.metadata_normalizer.MetadataNormalizer'.format(__SCRAPE_PACKAGE)
```
Please take a look at the next comment to understand where I wanna go with this. 😃

Reply:
good suggestion, thanks!